include/mesh_motion/FrameBase.h

```diff
@@ -45,6 +39,8 @@ class FrameBase
   bool is_inertial() const {
     return isInertial_; }
 
+  virtual void post_work() {};
```

Comment:
Remove semi-colon, will generate a warning with pedantic flags on.

Reply:
This has been fixed in ba40395